packages/aura-language-server/jest.config.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
       "js",
       "json"
     ],
-    setupTestFrameworkScriptFile: "<rootDir>/jest/matchers.ts",
+    setupFilesAfterEnv: ["jest-extended"],
```

Comment:
Just asking about this as I'm not sure whats going on.

Reply:
I upgraded the version of jest and this is the new way to specify `setupTestFrameworkScriptFile`, as this one takes more than one file. jest-extended is an extended set of matchers, one of which I used in my test and another one which replaces a custom matcher we were using